src/golang/lib/workflow/utils/database.go

```diff
@@ -255,7 +254,7 @@ func ReadLatestDAGFromDatabase(
 	dagRepo repos.DAG,
 	operatorReader operator.Reader,
 	artifactReader artifact.Reader,
-	workflowDagEdgeReader workflow_dag_edge.Reader,
+	dagEdgeReader repos.DAGEdge,
```

Comment:
@saurav-c should we name this `dagEdgeRepo`, so that it's consistent with other namings?